main_test.go

```
@@ -163,14 +164,129 @@ func TestPackageTemplateNoRenameDefaults(t *testing.T) {
+	}
+	return filepath.Join(dir, relDir, "linux-amd64")
+}
+
+// TestPackageRenameKeepsOldLinks pins the compatibility guarantee for a
+// --package-name rename: the packages ship under the new name, and every
+// filename an already-published URL could reference (the "latest" alias, the
+// versioned package, its checksum) still resolves via a symlink.
+func TestPackageRenameKeepsOldLinks(t *testing.T) {
+	out := runDoPackage(t, "aistor", "aistor")
+
+	real := []string{
```

golangci-lint (2.12.2)

[error] variable real has same name as predeclared identifier

(predeclared)